AutomatedTesting/Gem/PythonTests/atom_renderer/test_Atom_GPUTests.py

```diff
@@ -114,7 +169,7 @@ def test_AtomFeatureIntegrationBenchmark(
             TEST_DIRECTORY,
             editor,
             "hydra_GPUTest_AtomFeatureIntegrationBenchmark.py",
-            timeout=EDITOR_TIMEOUT,
+            timeout=600,
```

Comment:
check that cmakelists.txt timeout for this is at least greater than 1200 as this will run twice. also add room for other tests and buffer for initialization to cmakelists.txt